tests/appsec/test_appsec_utils.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+import pytest
+
+from ddtrace import constants
+import ddtrace.appsec.utils as utils
+from tests.utils import override_env
+
+
+@pytest.fixture(autouse=True)
+def reset_template_caches():
+    utils._HTML_BLOCKED_TEMPLATE_CACHE = None
+    utils._JSON_BLOCKED_TEMPLATE_CACHE = None
+    try:
+        yield
+    finally:
+        utils._HTML_BLOCKED_TEMPLATE_CACHE = None
+        utils._JSON_BLOCKED_TEMPLATE_CACHE = None
+
+
+def test_get_blocked_template_no_env_var_html():
+    with override_env(dict(DD_APPSEC_HTTP_BLOCKED_TEMPLATE_HTML="", DD_APPSEC_HTTP_BLOCKED_TEMPLATE_JSON="")):
+        assert utils._get_blocked_template("text/html") == constants.APPSEC_BLOCKED_RESPONSE_HTML
+
+
+def test_get_blocked_template_no_env_var_json():
+    with override_env(dict(DD_APPSEC_HTTP_BLOCKED_TEMPLATE_HTML="", DD_APPSEC_HTTP_BLOCKED_TEMPLATE_JSON="")):
+        assert utils._get_blocked_template("other") == constants.APPSEC_BLOCKED_RESPONSE_JSON
```

Comment:
parametrizing could clean up/reduce duplicated code a bunch.


mild suggestion, I don't have a strong opinion, just calling out the option.

Reply:
For simple and not checking many cases tests like this I prefer the version without indirections, I just can read them better.